Podfile

```diff
@@ -33,12 +33,12 @@ target 'AlphaWallet' do
   pod 'PaperTrailLumberjack/Swift'
   pod 'Charts'
   pod 'CocoaLumberjack', '3.7.0'
-  pod 'AlphaWalletAddress', :path => 'modules/AlphaWalletAddress'
-  pod 'AlphaWalletCore', :path => 'modules/AlphaWalletCore'
-  pod 'AlphaWalletGoBack', :path => 'modules/AlphaWalletGoBack'
-  pod 'AlphaWalletENS', :path => 'modules/AlphaWalletENS'
-  pod 'AlphaWalletOpenSea', :path => 'modules/AlphaWalletOpenSea'
-  pod 'AlphaWalletFoundation', :path => 'modules/AlphaWalletFoundation'
+  pod 'AlphaWalletAddress', :path => '../alpha-wallet-ios'
```

Comment:
Will this work too: `:path => '.'` work?

Reply:
will check

Reply:
yes, it works!, updating with using `'.'`